packages/angular_devkit/build_angular/src/angular-cli-files/utilities/service-worker/index.ts

```diff
@@ -13,6 +13,10 @@ import * as crypto from 'crypto';
 import * as fs from 'fs';
 import * as semver from 'semver';
 
+// @ts-ignore
+// this is due that there are no `@types/uglify-es`
```

Comment:
Note: will eventually remove it when this is done https://github.com/DefinitelyTyped/DefinitelyTyped/pull/27394